### PR TITLE
Changes to README for IBM Cloud, Flow, etc.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
 [![Build Status](https://travis-ci.org/IBM/watson-conversation-slots-intro.svg?branch=master)](https://travis-ci.org/IBM/watson-conversation-slots-intro)
-![Bluemix Deployments](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/badge.svg)
+![IBM Cloud Deployments](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/badge.svg)
 
 # Creating a Pizza ordering Chatbot using Watson Conversation Slots feature
 
-In this developer journey, we will use the Watson Conversation Slots feature to
+In this developer Code Pattern, we will use the Watson Conversation Slots feature to
```
Comment (markstur) [inline README.md]: I would drop the "developer" here for consistency, but it is not bad to keep it in here otherwise.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 * Use Conversation Slots to handle various inputs within one Node.
 
-![Flow](doc/source/images/slotsFlow.png)
+![](doc/source/images/architecture.png)
 
-### With Watson
+## Flow
 
-Want to take your Watson app to the next level? Looking to leverage Watson Brand assets? Join the [With Watson](https://www.ibm.com/watson/with-watson) program which provides exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
+1. User send messages to the application (running locally or on IBM Cloud)
```
Comment (markstur) [inline README.md]: s/send/sends/  or "Users send"?

Comment (markstur) [reply README.md]: and missing period

File: README.md [hunk 3]
```diff
@@ -29,35 +31,40 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
 
-#### Running this application in a container with Kubernetes on Bluemix
+#### Running this application in a container with Kubernetes on IBM Cloud
 
 [![](https://i.ytimg.com/vi/G-rESweRG84/0.jpg)](https://youtu.be/G-rESweRG84)
 
 # Steps
 
-[![Deploy to Bluemix](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/button.svg)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-conversation-slots-intro)
+## Deploy to Bluemix
```
Comment (markstur) [inline README.md]: Deploy to IBM Cloud

File: README.md [hunk 4]
```diff
@@ -29,35 +31,40 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
 
-Click the ``Deploy to Bluemix`` button and hit ``Create`` and then jump to step 5
+[![Deploy to IBM Cloud](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/button.svg)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-conversation-slots-intro)
+
+Click the ``Deploy to IBM Cloud`` button and hit ``Create`` and then jump to step 5
```
Comment (markstur) [inline README.md]: missing period here I guess. Not sure w/ the OR but the others use period.

File: README.md [hunk 5]
```diff
@@ -29,35 +31,40 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
 
  **OR** 
 
+## Run locally
  Perform steps 1-5:
 
 1. [Clone the repo](#1-clone-the-repo)
-2. [Create Bluemix services](#2-create-bluemix-services)
-3. [Get Bluemix credentials and add to .env](#3-get-bluemix-services-credentials-and-add-to-env-file)
+2. [Create IBM Cloud services](#2-create-bluemix-services)
```
Comment (markstur) [inline README.md]: broken link - header changed

File: README.md [hunk 6]
```diff
@@ -29,35 +31,40 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
+3. [Get IBM Cloud credentials and add to .env](#3-get-bluemix-services-credentials-and-add-to-env-file)
```
Comment (markstur) [inline README.md]: broken link

File: README.md [hunk 7]
```diff
@@ -223,17 +230,17 @@ Type reset to start again and test this by adding the phrase "to eat there...":
 * Deploy using Cloud Foundry 'cf push' gives:
 
 ``FAILED
-Could not find service <Watson_service> to bind to <Bluemix_application>``
+Could not find service <Watson_service> to bind to <IBM Cloud_application>``
```
Comment (markstur) [inline README.md]: space in IBM Cloud_application looks accidental(?) but doesn't hurt

File: README.md [hunk 8]
```diff
@@ -29,35 +31,40 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
 
  **OR** 
 
+## Run locally
  Perform steps 1-5:
 
 1. [Clone the repo](#1-clone-the-repo)
-2. [Create Bluemix services](#2-create-bluemix-services)
-3. [Get Bluemix credentials and add to .env](#3-get-bluemix-services-credentials-and-add-to-env-file)
+2. [Create IBM Cloud services](#2-create-IBM-cloud-services)
```
Comment (markstur) [inline README.md]: link doesn't work.  s/IBM/ibm/  2 places #2 and #3

**Comment by stevemart** [comment]
@scottdangelo lets work on getting a pretty version of the diagram

**Pull request merged**